test/integration/generated/optionalnull/src/optionalNullClient.ts

```diff
@@ -39,6 +40,29 @@ export class OptionalNullClient extends coreClient.ServiceClient {
       baseUri: options.endpoint ?? options.baseUri ?? "{$host}"
     };
     super(optionsWithDefaults);
+
+    if (options?.pipeline && options.pipeline.getOrderedPolicies().length > 0) {
+      const pipelinePolicies: coreRestPipeline.PipelinePolicy[] = options.pipeline.getOrderedPolicies();
```

Comment:
```suggestion
      const pipelinePolicies = options.pipeline.getOrderedPolicies();
```

I don’t think this change is necessary btw. If you find it preferable, go for it! — I’m just tying to make sure I don’t miss any feedback.

Reply:
Thanks for the suggestion. Will do it in the next PR. Don't want to stop release because of the nit change :-)